internal/provider/pipeline_resource.go

```diff
@@ -8791,3 +8869,75 @@ func (r *PipelineResource) Delete(ctx context.Context, req resource.DeleteReques
 func (r *PipelineResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
 	resp.Diagnostics.Append(resp.State.SetAttribute(ctx, path.Root("id"), req.ID)...)
 }
+
+// Etleap monkey-patch functions
+func waitPipelineRenamed(ctx context.Context, r *PipelineResource, pipelineId string, destinationId string, originalSchema string, originalTable string, newSchema string, newTable string) (*operations.GetPipelineRequest, error) {
+	const (
+		timeout = 75 * time.Minute
```

Comment:
Do we expect this to take 75 minutes? Maybe we set it to something more reasonable, like 10-15 minutes?

Reply:
Sure, made it 15 minutes.